modules/home-manager/gtk.nix

```diff
@@ -79,5 +84,25 @@ in
         "gtk-4.0/gtk.css".source = "${gtk4Dir}/gtk.css";
         "gtk-4.0/gtk-dark.css".source = "${gtk4Dir}/gtk-dark.css";
       };
+
+    home.packages = lib.mkIf cfg.gnome [pkgs.gnomeExtensions.user-themes];
+
+    dconf.settings = lib.mkIf cfg.gnome {
```

Comment:
we should assert that dconf is enabled, similar to xdg above

Reply:
I figured that since this is a theme for gnome, dconf would be enabled anyway but yes I can add that for good measure

Reply:
Actually dconf is enabled in the nixos config not in home-manager so I have no idea of how to assert that it is enabled in a home-manager module

Reply:
ah you're right, i forgot home-manager can't enable it. sorry about that :p